From 53d8eabe3df36015daf40a7a9bfad9b2ffafc6bd Mon Sep 17 00:00:00 2001
From: Alex Deucher <alexander.deucher@amd.com>
Date: Mon, 21 Dec 2015 17:07:40 -0500
Subject: amd/powerplay: disable powerplay by default initially

Hopefully we can enable this by default once we get more
upstream feedback on stability, etc.

Signed-off-by: Alex Deucher <alexander.deucher@amd.com>

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_powerplay.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_powerplay.c
index ddb90eb..5ee9a06 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_powerplay.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_powerplay.c
@@ -101,7 +101,7 @@ static int amdgpu_pp_early_init(void *handle)
 	switch (adev->asic_type) {
 		case CHIP_TONGA:
 		case CHIP_FIJI:
-			adev->pp_enabled = (amdgpu_powerplay == 0) ? false : true;
+			adev->pp_enabled = (amdgpu_powerplay > 0) ? true : false;
 			break;
 		default:
 			adev->pp_enabled = (amdgpu_powerplay > 0) ? true : false;
-- 
cgit v0.10.2

